server/services/team/service.ts

```diff
@@ -518,28 +518,28 @@ export class TeamService extends ServiceClass implements ITeamService {
 			};
 		}
 
-		const cursor = this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset, query);
+		const users = await this.Users.find({ ...query }).toArray();
```

Comment:
Since the `query` is an `options` param, shouldn't this line be `Users.find({}, options)`? 🤔